src/algebra/group_ring_action.lean

```diff
@@ -40,7 +40,7 @@ class mul_semiring_action (M : Type u) (R : Type v) [monoid M] [semiring R]
 
 section semiring
 
-variables (M G : Type u) [monoid M] [group G]
+variables (M G : Type*) [group G] [monoid M]
```

Comment:
Why did this line change?

Reply:
I was trying out Floris' suggestion. Otherwise `N` and `M` can't both be implicit at the same time. ( I could only make `N` implicit) but I can change it but if this doesn't work.

Reply:
* I could only make `N` implicit

Reply:
I don't see how swapping the order of the typeclass arguments could be relevant here

Reply:
What probably makes the most sense here is
```suggestion
variables (M N G : Type*) [monoid M] [monoid N] [group G]
```
Then you can avoid repeating the typeclasses below